app/services/acuant/acuant_client.rb

```diff
@@ -50,5 +74,36 @@ def merge_facial_match_and_liveness_response(facial_match_response, liveness_res
         },
       )
     end
+
+    def check_results(post_response, instance_id)
+      if post_response.success?
+        fetch_doc_auth_results(instance_id)
+      else
+        failure(post_response.errors.first, post_response.to_h)
+      end
+    end
+
+    def fetch_doc_auth_results(instance_id)
+      results_response = get_results(instance_id: instance_id)
+      handle_document_verification_failure(results_response) unless results_response.success?
+
+      results_response
+    end
+
+    def handle_document_verification_failure(get_results_response)
+      mark_step_incomplete(:document_capture)
+      extra = get_results_response.to_h.merge(
+        notice: I18n.t('errors.doc_auth.general_info'),
+      )
+      failure(get_results_response.errors.first, extra)
```

Comment:
Same here, couldn't we return `get_results_response` directly?

Reply:
Not if we want the extra to include our custom text. Or are you saying that we can add our notice text into the response from Acuant?